### DISCUSSION TITLE
How get back up the data regularly

### DISCUSSION THREAD

**Discussion by code-blooded-guy** [Sunbird-RC/community#287] [Show & Tell]
How can we have a regular data backup? preferably daily or weekly?
We have the Production server where we need the backup of data every 5 or 10 minutes.
Can anyone please help us with the same or someone has already done it?

**Comment by parthlawate** [comment]
@tejash-jl can you help with this pls ? cc @Renuka-S @jayadinni

**Reply by tejash-jl** [reply]
@code-blooded-guy,
- The simplest approach is to run cronjob if it's weekly or daily (during low scale time)
- If you require a minute wise backup, then you can enable Postgres WAL and backup those WAL files periodically. This will help you to do PITR (Point-In-Time-Recovery)
